lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -59,6 +59,8 @@ abstract class EngineInputType {
   /// <https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode>.
   String get inputmodeAttribute;
 
+  bool get submitActionOnEnter => true;
```

Comment:
May be we can also add one line of comment.

Reply:
Ah yes. Thanks for the reminder.